README.md

```diff
@@ -1,42 +1,41 @@
-![](https://raw.githubusercontent.com/balena-io-projects/balena-sound/master/images/balenaSound-logo.png)
+# ![logo](./docs/images/balenaSound-logo.png)
```

Comment:
let's keep the logo the old way, with just the alt. Fixing this should fix the build.

Reply:
just pushed the changes